### PR TITLE
Goce Lazarevski 5911

### PR DESCRIPTION
**Pull request by LazarevskiGoce16**

### CODE REVIEW

File: code/5911-goce-lazarevski/script.js [hunk 1]
```diff
@@ -1,3 +1,156 @@
...
+    displayListOfEpisodes(data);
+    populateFilters(data);
+    
+    document.querySelector("#name-filter").addEventListener("input", () => applyFilters(data));
+    document.querySelector("#era-filter").addEventListener("change", () => applyFilters(data));
+    document.querySelector("#doctor-filter").addEventListener("change", () => applyFilters(data));
+    document.querySelector("#companion-filter").addEventListener("change", () => applyFilters(data));
+    
+    document.querySelectorAll(".header-cell").forEach(header => {
+        header.dataset.order = "asc";
```
Comment (sweko) [inline code/5911-goce-lazarevski/script.js:141]: This code only executes once 😄 Otherwise it is great

File: code/5911-goce-lazarevski/script.js [hunk 2]
```diff
@@ -1,3 +1,156 @@
...
+        const matchesDoctor = !doctorFilter || episode.doctor.actor === doctorFilter;
+        const matchesCompanion = !companionFilter || (episode.companion && episode.companion.actor === companionFilter);
+        return matchesName && matchesEra && matchesDoctor && matchesCompanion;
+    });
+    
+    displayListOfEpisodes(filteredEpisodes);
+};
+
+const sortEpisodes = (data, key, order) => {
+    return data.slice().sort((a, b) => {
```
Comment (sweko) [inline code/5911-goce-lazarevski/script.js:105]: nice approach

**Review by sweko — APPROVED** [review]
Great work overall - but try to figure out why the sort is broken.
@LazarevskiGoce16 - 100/100 points

File: code/5911-goce-lazarevski/script.js (hunk 1 above)
Comment (LazarevskiGoce16) [reply code/5911-goce-lazarevski/script.js:141]: Yeah, it need to be refreshed to work for the other functionalities, I will try to fix it and commit the new version.

File: code/5911-goce-lazarevski/script.js (hunk 2 above)
Comment (LazarevskiGoce16) [reply code/5911-goce-lazarevski/script.js:105]: thanks

**Pull request merged**